reviews/models/mixins.py

```diff
@@ -265,3 +261,16 @@ def reviews_notification(self, context, notify_submit=False):
                     node_lineage=node_lineage_ids
                 )
                 digest.save()
+
+# Handle email notifications for a new submission.
+@reviews_signals.reviews_email_submit.connect
+def reviews_submit_notification(self, context):
+    template = ''.join(context.get('template'))
+    for user_id in context.get('email_recipients'):
```

Comment:
If a key will always be in a dictionary, if the key missing would be a programming error, I'd prefer to use `[]` instead of `get` -- the exception raised is much easier to understand/debug than when it tries to do something with `None` later on.

If the key could reasonably be missing without error, then use `get` and handle the possible `None`.